src/content/docs/experiment/link-element.mdx

```diff
@@ -0,0 +1,141 @@
+---
+title: Monetization <link> element
+tableOfContents:
+    maxHeadingLevel: 4
+---
+
+import { LinkOut, Hidden } from "@interledger/docs-design-system";
+import Specification from '/src/components/docs/Specification.astro'
+import BrowserCompat from '/src/components/docs/BrowserCompat.astro'
+import data from '/src/data/browser-compat-data/link.json'
+import { Steps, Tabs, TabItem, Badge } from '@astrojs/starlight/components';
+
+<Badge text="HTML DOM API" style="position: relative; top:-30px;" />
+
+From a creator/developer perspective, the monetization `<link>` element is one of the two key pieces to web monetizing a page. The second piece is a wallet address/payment pointer.
+
+## Prerequisites
+
+To web monetize an HTML document, such as a web page:
+* You must have an account with a [compatible wallet](/resources/wallets)
+* You must have the wallet address, or payment pointer in <LinkOut href="https://paymentpointers.org">URL format</LinkOut>, from your wallet provider
+* Your document must be secure and served over HTTPS
+
+## Monetization `<link>` element
+
+The monetization `<link>` element is what indicates your HTML page supports Web Monetization.
```

Comment:
Should use "HTML page" vs "HTML document" consistently across docs?

Reply:
Updated in user docs, left in specification